src/components/FormBuilder.tsx

```diff
@@ -280,7 +287,7 @@ export class FormBuilder extends React.Component<IFormBuilderProps, {}> {
 
         const component = React.createElement(fieldDef.builder, fieldBuilderProps);
         return (
-            <div className='form-builder-field' key={field.id}>
+            <div ref={this.attachFieldRef} className='form-builder-field' key={field.id}>
```

Comment:
I'll merge this PR and if we need a different UX I'll implement this using editingFieldID